auth/src/main/java/no/nav/common/auth/oidc/filter/OidcAuthenticatorConfig.java

```diff
@@ -38,6 +38,12 @@ public class OidcAuthenticatorConfig {
     // Url to call when refreshing the id token (optional)
     public String refreshUrl;
 
+    // Domain for id token cookie (optional)
+    public String cookieDomain;
+
+    // Path for id token cookie (optional)
+    public String cookiePath;
```

Comment:
Burde det vært spesifisert at disse 2 gjelder kun for id tokens som blir opprettet av refreshing?
Det er åpenbart hvis man er kjent med auth-modulen, men kanskje litt mindre forståelig for folk som ikke er like kjent.

Reply:
Skader ikke å være tydelig.
```java
    // Domain to use for the refreshed id token cookie (optional)
    public String refreshedCookieDomain;

    // Path to use for the refreshed id token cookie (optional)
    public String refreshedCookiePath;
```